libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/SPRequestHandler.java

```diff
@@ -129,6 +129,9 @@ private SPConfig buildSPConfig(String loginUrl, String userHint) {
 
     private void handleError(String error) {
         SalesforceSDKLogger.e(TAG, "Error received from IDP app: " + error);
+        if (authCallback != null) {
```

Comment:
Minor unrelated bug in the IDP flow where the error wasn't getting displayed as a `Toast`.